2016-08-15 22:17:56.056 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:19:19.152 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:20:15.731 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:24:47.996 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:25:56.813 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:26:17.267 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:27:12.344 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:27:58.580 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:29:15.074 [Druid-ConnectionPool-Create-942986815] ERROR com.lfx.druid.model.DruidDataSource - create connection error, url: jdbc:mysql://192.168.3.6:3306/test?useUnicode=true&characterEncoding=UTF-8, errorCode 0, state 08S01
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_40]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_40]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_40]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_40]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1116) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:344) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2332) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2369) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2153) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:792) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47) ~[mysql-connector-java-5.1.18.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_40]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_40]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_40]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_40]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:381) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:305) ~[mysql-connector-java-5.1.18.jar:na]
	at com.lfx.druid.model.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:132) ~[classes/:na]
	at com.lfx.druid.model.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:109) ~[classes/:na]
	at com.lfx.druid.model.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:240) ~[classes/:na]
Caused by: java.net.ConnectException: Operation timed out
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_40]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:244) ~[na:1.8.0_40]
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:257) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:294) ~[mysql-connector-java-5.1.18.jar:na]
	... 15 common frames omitted
2016-08-15 22:30:30.119 [Druid-ConnectionPool-Create-942986815] ERROR com.lfx.druid.model.DruidDataSource - create connection error, url: jdbc:mysql://192.168.3.6:3306/test?useUnicode=true&characterEncoding=UTF-8, errorCode 0, state 08S01
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_40]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_40]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_40]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_40]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1116) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:344) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2332) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2369) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2153) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:792) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47) ~[mysql-connector-java-5.1.18.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_40]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_40]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_40]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_40]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:381) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:305) ~[mysql-connector-java-5.1.18.jar:na]
	at com.lfx.druid.model.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:132) ~[classes/:na]
	at com.lfx.druid.model.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:109) ~[classes/:na]
	at com.lfx.druid.model.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:240) ~[classes/:na]
Caused by: java.net.ConnectException: Operation timed out
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_40]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:244) ~[na:1.8.0_40]
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:257) ~[mysql-connector-java-5.1.18.jar:na]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:294) ~[mysql-connector-java-5.1.18.jar:na]
	... 15 common frames omitted
2016-08-15 22:31:08.883 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource closed
2016-08-15 22:31:52.787 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:31:55.448 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource closed
2016-08-15 22:37:55.356 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:37:57.541 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource closed
2016-08-15 22:44:10.176 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:44:12.209 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource closed
2016-08-15 22:46:00.247 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource inited
2016-08-15 22:46:02.843 [main] INFO  com.lfx.druid.model.DruidDataSource - {dataSource closed
2016-08-15 22:56:18.261 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 22:56:20.930 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
2016-08-15 23:00:23.808 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 23:00:25.883 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
2016-08-15 23:05:03.927 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 23:05:05.957 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
2016-08-15 23:05:18.176 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 23:05:20.139 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
2016-08-15 23:25:31.168 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 23:25:32.699 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
2016-08-15 23:26:59.718 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource inited
2016-08-15 23:27:01.191 [main] INFO  com.lfx.druid.model.DruidDataSource - dataSource closed
